Dshop/apps/products_catalogue/models.py

```diff
@@ -27,3 +27,17 @@ class Meta:
         verbose_name = 'Category'
         verbose_name_plural = 'Categories'
 
+
+class Product(models.Model):
```

Comment:
IMO the Procuct class should inherit from class CatalogueItemModel as same as Category class, we can use the same fields and slug generating method.

Reply:
good catch!

Reply:
So, I should remove `name` from Product class if this class inherit this from CatalogueItemModel , right?

Reply:
yes. name, slug and is_active are already in CatalogueItemModel.